resourcematch-app/src/components/TextInputForm.tsx

```diff
@@ -0,0 +1,98 @@
+import { Formik, Form, Field, FormikHelpers, ErrorMessage, FieldInputProps } from 'formik'
+import styled from 'styled-components'
+import { useState } from 'react'
+
+const StyledForm = styled(Form)`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-top: 20px;
+`
+
+const InputContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 10px;
+`
+
+const Button = styled.button`
+  align-items: center;
+  margin-bottom: 10px;
+`
+
+const InputLabel = styled.label`
+  text-align: center;
+  margin-bottom: 5px;
+`
+
+const InputField = styled.input`
+  max-width:800px;
+  max-height:800px;
+  min-width: 600px;
+  max-height: 300px;
+  min-height: 100px;
+  padding: 8px;
+`
+
+const StyledErrorMessage = styled(ErrorMessage)`
+  color: red;
+  text-align: center;
+  `
+
+
+interface TextInputFormProps {
+  onSubmit: (input: string) => void
```

Comment:
Minor: I think the return type here should be `Promise<void>` to indicate async functions.